torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -61,16 +64,30 @@ void ProfilingRecord::insertShapeProfile(Node* n, Value* i) {
   n->replaceInputWith(i, pn->output());
 }
 
+bool isProfilable(Node* n) {
+
+  if (IsFusibleOperation(n) || isSupported(n)) {
```

Comment:
If we want to support both the old and the new fuser, then this check should be more selective: i.e. we should only use one of these conditions depending on what fuser is enabled.

But I'm not sure there is a value in supporting the old one at this point at all.

Reply:
agree wholeheartedly. I'm going to clean up this PR, this was very useful for testing.